src/battle_script_commands.c

```diff
@@ -10180,6 +10226,12 @@ static void Cmd_various(void)
             gBattlescriptCurrInstr = cmd->nextInstr;   // can heal
         return;
     }
+    case VARIOUS_REMOVE_WEATHER:
+    {
+        VARIOUS_ARGS();
+        RemoveAllWeather();
+        break;
+    }
```

Comment:
Please use a callnative instead. Also I think you forgot to go to the next instruction.

Reply:
I just copied VARIOUS_REMOVE_TERRAIN, and that doesn't go to the next instruction.
I'll move it to callnative though

Reply:
That is weird. I wonder if there is a bug.

Reply:
oh it is all fine. The next instruction call is at the end of the function. I'm just callnative pilled.